selftests/functional/basic.py

```diff
@@ -98,6 +98,14 @@ def test(self):
         logging.getLogger("some.other.logger").info("SHOULD BE ON debug.log")
 """
 
+EXEC_ENV_VARIABLE_TEST = """#!/bin/bash
+if [[ -z "${TEST_ENV}"  ]]; then
+    exit 1
```

Comment:
So, this script "fails" when the variable is empty, although that's actually the "good" case below.  IIUC, that's handled by expecting a return code of 1 below, but it still seems a little counterintuitive to me to do this rather than having the script fail if the variable is set.

Reply:
Yes, that is true. I will update that.